src/main.ts

```diff
@@ -33,6 +33,27 @@ async function main() {
     const currentAppProperty = await client.containerApps.get(taskParams.resourceGroup, taskParams.containerAppName);
 
     // TBD: Remove key when there is key without value
+
+    let currentProductionRevisionName: string | undefined = '';
+    currentAppProperty.configuration!.ingress!.traffic!.forEach((traffic: TrafficWeight) => {
+      if (traffic.weight == 100) {
+        currentProductionRevisionName = traffic.revisionName!;
+      }
+    });
+
+    const traffic = [
```

Comment:
In this case, if we are developing as a team and multiple people send pull requests and push, what will happen?